pkg/resource/aws/aws_route53_health_check_test.go

```diff
@@ -0,0 +1,93 @@
+package aws_test
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/route53"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/cloudskiff/driftctl/pkg/analyser"
+	awsresources "github.com/cloudskiff/driftctl/pkg/resource/aws"
+	"github.com/cloudskiff/driftctl/test/acceptance"
+	"github.com/cloudskiff/driftctl/test/acceptance/awsutils"
+	"github.com/r3labs/diff/v2"
+)
+
+func TestAcc_AwsRoute53HealthCheck(t *testing.T) {
+	var mutatedHealthCheckID string
+	acceptance.Run(t, acceptance.AccTestCase{
+		Path: "./testdata/acc/aws_route53_health_check",
+		Args: []string{"scan", "--filter", "Type=='aws_route53_health_check'"},
+		Checks: []acceptance.AccCheck{
+			{
+				Env: map[string]string{
+					"AWS_REGION": "us-east-1",
+				},
+				Check: func(result *acceptance.ScanResult, stdout string, err error) {
+					if err != nil {
+						t.Fatal(err)
+					}
+					result.AssertDriftCountTotal(0)
+					result.AssertDeletedCount(0)
+					result.AssertManagedCount(2)
+
+					for _, resource := range result.Analysis.Managed() {
```

Comment:
to prevent crash if there is no managed resource. I was not sure the assertions above stopped the run. this is not so costly as there is 2 resourcea anyway...

Reply:
not costly at all but difficult to understand why 😅 , you can safely the test will fail immediately if the assertion failed

Reply:
![image](https://user-images.githubusercontent.com/6154987/108036686-d6411100-7038-11eb-962b-8159d256c916.png)